src/key_gen/mod.rs

```diff
@@ -756,9 +790,14 @@ impl<S: SecretId> KeyGen<S> {
             .count()
     }
 
-    /// Returns `true` if enough parts are complete to safely generate the new key.
-    pub fn is_ready(&self) -> bool {
-        self.complete_parts_count() >= self.threshold
+    // Returns `true` if all parts are complete to safely generate the new key.
+    fn is_ready(&self) -> bool {
+        self.complete_parts_count() == self.pub_keys.len()
+    }
```

Comment:
Does this mean that we now need full participation?

Reply:
I think we already claimed so? 
I mean for the `final` participants. Those unresponsive nodes will be kicked out during the process.

Reply:
OK, just making sure I understand it.